bears/general/ImageDimensionBear.py

```diff
@@ -0,0 +1,70 @@
+import os
+import yaml
+import shutil
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from sarge import run, Capture
+
+
+class ImageDimensionBear(GlobalBear):
+    """
+    Checks the dimension of an image us a gem.
+
+    More information is available at <github.com/Abhi2424shek/img_checker>
+    """
+
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {GemRequirement('img_checker'),
+                    PipRequirement('pyyaml', '3.12')}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Image Dimension'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('img_checker') is None:
+            return 'img_checker is not installed.'
+        return True
+
+    def run(self,
+            image_file: str='',
+            width: int=0,
+            height: int=0):
+        """
+        This bear ensures that images used in your project are
+        within a fixed dimension.
+
+        WARNING: If your repository has an existing img_config.yml
+        file, please change it since this bear creates a file
+        with that name and removes it in the end.
+
+        :param image_file: The file/directory that bear
+                           will look for images
+        :param width:      The maximum width value allowed
+                           for images
+        :param height:     The maximum height value allowed
+                           for images
+        """
+        with open('img_config.yml', 'w') as yaml_file:
+            config = [{'directory': image_file,
+                       'width': width,
+                       'height': height}]
+            yaml.dump(config, yaml_file, default_flow_style=False)
+        cmd = 'img_checker'
+        output = run(cmd, stdout=Capture(), stderr=Capture())
+        if (output.returncode):
+            lines = output.stdout.text.split('\n')[1:-2]
+            for line in lines:
+                if '.png' in line:
+                    fileName = line[10:line.index('.png')+4]
```

Comment:
didn't know this but will do the change, first let me fix the problem with the test fail in appveyor. also if possible please help me understand the error,